syncthing-client/src/main/kotlin/net/syncthing/java/client/SyncthingClient.kt

```diff
@@ -35,6 +35,21 @@ import java.util.concurrent.Executors
 import java.util.concurrent.TimeUnit
 import java.util.concurrent.atomic.AtomicBoolean
 import kotlin.collections.ArrayList
+import kotlin.collections.HashMap
+import kotlin.collections.List
+import kotlin.collections.component1
+import kotlin.collections.component2
+import kotlin.collections.distinctBy
+import kotlin.collections.emptySet
+import kotlin.collections.filterNot
+import kotlin.collections.filterNotNull
+import kotlin.collections.find
+import kotlin.collections.forEach
+import kotlin.collections.groupBy
+import kotlin.collections.map
+import kotlin.collections.mutableListOf
+import kotlin.collections.set
+import kotlin.collections.takeWhile
```

Comment:
These all don't need to be imported explicitly.

Reply:
Interestingly, after running the "optimize imports" in Android Studio again, these entries were removed.